Fix/odd 674 675 677 658 659

deoyani opened the pull request:
This pull request addresses issues listed in following JIRA ticket numbers.
[ODD-674](http://mml.nist.gov:8080/browse/ODD-674), [ODD-675](http://mml.nist.gov:8080/browse/ODD-675),[ODD-677](http://mml.nist.gov:8080/browse/ODD-677),
[ODD-658](http://mml.nist.gov:8080/browse/ODD-658), [ODD-659](http://mml.nist.gov:8080/browse/ODD-659)

Summary of the changes in this branch is as below.
Updated Data access section with the requirement to link to homepage.
Updated spacing in Data Access section
Changed color of visit home page
Fixed issue with files count in Files section
Fixed reference section to avoid multiple new lines
Fixed breaking identifier section
Updated citation with header color and fonts
Updated fonts everywhere under Description section.
Found issue with nginx proxy server and also removed pdr for /about page
Fixed issue with the url with arkid

RayPlante commented:
These latest changes result in the following build error: 
```
ERROR in src/app/landing/error.component.ts.ErrorComponent.html(6,12): : 
Property 'errorcode' does not exist on type 'ErrorComponent'.
```

deoyani commented:
ODD-674 (data access section):
  updated with suggested changes.
ODD-675 (breaking identifier): 
  Not able to reproduce may be specific to browser and OS.
ODD-659 (citation rendering): Updated url for doi.
ODD-680: Update error page with response http code.

deoyani commented:
ODD-674 (data access section):
 Updated Restricted access section with spacing.
ODD-680 (404 when not found): 
 Updated Error Handling. 
ODD-675 (breaking identifier): Redesigned page grid. Issue was is browser is zoomed in arid identifier would break and go on new line. Recreated div structure to avoid this.
